--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -404,7 +404,7 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
-      return((ptrbits == 64) ? "-m64" : "-m31");
+      return(sp);
    if (OS == OSAIX)
       return((ptrbits == 64) ? "-maix64" : "-maix32");
 
@@ -801,8 +801,6 @@ int main(int nargs, char **args)
             fprintf(fpout, "_fbsd");
       }
    }
-   if (MachIsS390(mach))
-      fprintf(fpout, ptrbits == 32 ? "-m31" : "-m64");
    fprintf(fpout, "\n   F77SYSLIB = %s\n", f77lib ? f77lib : "");
    fprintf(fpout, "   BC = $(KC)\n");
    fprintf(fpout, "   NCFLAGS = $(KCFLAGS)\n");
--- a/CONFIG/src/probe_comp.c
+++ b/CONFIG/src/probe_comp.c
@@ -598,7 +598,7 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
-      return((ptrbits == 64) ? "-m64" : "-m31");
+      return(sp);
    if (OS == OSAIX)
       return((ptrbits == 64) ? "-maix64" : "-maix32");
 
