--- a/CONFIG/include/atlconf.h
+++ b/CONFIG/include/atlconf.h
@@ -77,6 +77,14 @@ enum MACHTYPE {MACHOther, IbmPwr3, IbmPw
 #define MachIsALPHA(mach_) \
   ( 0 )
 #endif
+#ifdef __ARMEL__
+#define MachIsARMEL(mach_) \
+  ( __ARMEL__ )
+#else
+#define MachIsARMEL(mach_) \
+  ( 0 )
+#endif
+
 
 static char *f2c_namestr[5] = {"UNKNOWN","Add_", "Add__", "NoChange", "UpCase"};
 static char *f2c_intstr[5] =
--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -395,6 +395,8 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
      return(sp);
    if (MachIsALPHA(arch))
      return(sp);
+   if (MachIsARMEL(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
--- a/CONFIG/src/probe_comp.c
+++ b/CONFIG/src/probe_comp.c
@@ -588,6 +588,8 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
      return(sp);
    if (MachIsALPHA(arch))
      return(sp);
+   if (MachIsARMEL(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
