--- a/CONFIG/include/atlconf.h
+++ b/CONFIG/include/atlconf.h
@@ -83,7 +83,13 @@ enum MACHTYPE {MACHOther, PPCG4, PPCG5,
    ( (mach_) >= ARM64xg && || (mach_) <= ARM64a57)
 #define MachIsS390(mach_) \
    ( (mach_) >= IbmZ9 && (mach_) <= IbmZ13 )
-
+#ifdef __hppa__
+#define MachIsHPPA(mach_) \
+   ( __hppa__ )
+#else
+#define MachIsHPPA(mach_) \
+   ( 0 )
+#endif
 
 static char *f2c_namestr[5] = {"UNKNOWN","Add_", "Add__", "NoChange", "UpCase"};
 static char *f2c_intstr[5] =
--- a/CONFIG/src/SpewMakeInc.c
+++ b/CONFIG/src/SpewMakeInc.c
@@ -401,6 +401,8 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
 
    if (MachIsIA64(arch))
       return(sp);
+   if (MachIsHPPA(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
--- a/CONFIG/src/probe_comp.c
+++ b/CONFIG/src/probe_comp.c
@@ -595,6 +595,8 @@ char *GetPtrbitsFlag(enum OSTYPE OS, enu
 
    if (MachIsIA64(arch))
       return(sp);
+   if (MachIsHPPA(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (MachIsS390(arch))
